src/popup/router/components/SubheaderButton.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <Button
+    :class="['button', fill ]"
+    :to="to"
+    @click="$emit('click')"
+  >
+    <div class="box">
+      <span class="icon">
+        <slot />
+      </span>
+
+      <div class="message">
+        <span class="heading"> {{ header }} </span>
+        <span class="description"> {{ subheader }} </span>
+      </div>
+    </div>
+  </Button>
+</template>
+
+<script>
+export default {
+  props: {
+    to: { type: [String, Object], default: '' },
+    header: { type: String, default: '' },
+    subheader: { type: String, default: '' },
+    fill: {
+      type: String,
+      validator: (value) => ['primary', 'alternative'].includes(value),
+      default: 'primary',
+    },
+    isExternalLink: { type: Boolean },
+  },
+};
+</script>
+<style lang="scss" scoped>
```

Comment:
Please add empty line between script and style :)

Reply:
Hard to remember fine rules. Is it something we can add to the linter rules?